src/cmd/apply.ts

```diff
@@ -118,11 +118,20 @@ export const module: CommandModule = {
   command: cmdName,
   describe: 'Apply all, or supplied, k8s resources',
   builder: (parser: Argv): Argv => helmOptions(parser),
-
   handler: async (argv: HelmArguments): Promise<void> => {
     setParsedArgs(argv)
     setup()
     await prepareEnvironment()
-    await apply()
+    try {
+      await apply()
+    } catch (error) {
+      if (error.exitCode === 1) {
+        const d = terminal(`cmd:${cmdName}:k8s:events`)
+        const res = await nothrow($`kubectl get events -A --sort-by='.lastTimestamp'`)
```

Comment:
This can be pretty exhaustive right? Would be best to filter on namespace, but perhaps one of these is an easier solution?
- `--field-selector='reason!=info,reason!=Completed,reason!=Pulled,reason!=Created,reason!=Started,reason!=Scheduled,reason!=SuccessfulCreate'`
- `--field-selector='type!=Normal'`
- `|(head -n1 && tail -n30) `

Reply:
True, but I do not want to redact info about completed events. If you filter them out then it is hard to analyse the root cause chart installation failure. 
Perhaps obtaining events from last 1200s  (this is helmfile timeout) could limit the number of event.